cal/cal/views.py

```diff
@@ -139,3 +140,9 @@ def google_auth(request):
     else:
         auth_uri = flow.step1_get_authorize_url()
         return HttpResponseRedirect(auth_uri)
+
+def handler404(request):
+    response = render_to_response('404.html', {},
```

Comment:
`render_to_response` [is "likely to be deprecated in the future"](https://docs.djangoproject.com/en/1.10/topics/http/shortcuts/#render-to-response)